tests/conftest.py

```
@@ -5,5 +5,13 @@
 @pytest.fixture
 def engine():
     engine = sa.create_engine("crate://")
+    with engine.connect() as conn:
+        # Clean up stale "read-only" mode states.
+        try:
+            conn.execute(sa.text('ALTER TABLE testdrive.foo SET ("blocks.write"=false);'))
+        except Exception:  # noqa: S110
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)